README.md

```diff
@@ -1 +1,21 @@
-# MediaMonks - Frontend Coding Standards
+# frontend-coding-standards
+
+## Recommended Frameworks
+
+### React
+We recommend using [React](https://reactjs.org/) for 'big build like' Single Page Applications (SPA's). React is suited for long term projects that need stable and maintainable code. React works great together with [TypeScript](https://www.typescriptlang.org/).
+
+##### Getting started
```

Comment:
Do we recommend using CRA over [react-skeleton](https://github.com/hjeti/react-skeleton)?

Reply:
IMHO yes - react-skeleton is outdated, opts in with redux, etc.

Conclusion from last meetings was to start using CRA templates for different types of skeletons (bare bones with some tooling setup, or more integrated ones with specific state management, router setup, etc).

Same would apply for Vue with the new vue-cli IMO.

Reply:
Thanks for the clarification.